### PR TITLE
Edit Install MySQL Server on Ubnutu article

### LINKED ISSUE #1177: Audit Install MySQL server on Ubuntu
Install MySQL server on Ubuntu was created in 2011 and should probably have the commands checked. At the very least, it should have version numbers added. Also, the tone is pretty conversational.

https://support.rackspace.com/how-to/installing-mysql-server-on-ubuntu/ 

- [x] SME check
- [ ] Editor Check
- [ ] Update the 'audit_date' and 'last_modified_by' metadata
- [ ] If any changes were made, update the 'last_modified_date' metadata

### PR DESCRIPTION
**Pull request by kylelaffoon**
Fixes #1177 

Applied edits to Install MySQL Server on Ubuntu article.

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-56d0e3223c/how-to/)

File: content/cloud-servers/installing-mysql-server-on-ubuntu.md [hunk 1]
```diff
@@ -249,22 +217,19 @@ running a query to list all databases:
...
 ### Add a database user
 
-It's not a good idea to have applications connecting to the database
-using the root user. That gives them more privileges than they need.
-We'll create a user named "demouser" that applications can use to
-connect to the new database.
+When applications connect to the database using the root user, they usually have more privileges than they need. You can that applications can use to connect to the new database. In the following example, a user named **demouser** is created.
 
-To make the user run the following in the mysql shell:
+1. To creat a new user, run the following command in the `mysql` shell:
```
Comment (catlook) [inline content/cloud-servers/installing-mysql-server-on-ubuntu.md]: To greate a new ....

File: content/cloud-servers/installing-mysql-server-on-ubuntu.md [hunk 2]
```diff
@@ -5,170 +5,146 @@ title: Install MySQL Server on Ubuntu
...
 
 ### Install MySQL
 
-The easiest way to install the MySQL server is through the Ubuntu
-package manager:
+1. Install the MySQL server by using the Ubuntu package manager:
 
     sudo apt-get update
-    sudo apt-get install mysql-server
+    sudo apt-get install mysql-
```
Comment (catlook) [inline content/cloud-servers/installing-mysql-server-on-ubuntu.md]: should "install mysql-" have the hyphen at the end?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b247473c37/how-to/)

**Comment by catlook** [comment]
👍

**Pull request merged**